outboxrepo/migrations/000001_create_foundation_outbox_events.up.sql

```diff
@@ -1,8 +1,10 @@
 CREATE TABLE foundation_outbox_events (
-    id BIGINT PRIMARY KEY,
+    id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
     topic TEXT NOT NULL,
     key TEXT NOT NULL,
+    proto_name TEXT NOT NULL,
```

Comment:
We already have the proto name in headers. The outbox is designed to not know anything beyond the actual Kafka message bytes and corresponding headers.

Reply:
Done, now using headers